src/libraries/System.Net.Security/src/System/Net/Security/SslClientAuthenticationOptions.cs

```diff
@@ -73,5 +73,11 @@ public SslProtocols EnabledSslProtocols
         /// Use extreme caution when changing this setting.
         /// </summary>
         public CipherSuitesPolicy? CipherSuitesPolicy { get; set; }
+
+        /// <summary>
+        /// Specifies X509ChainPolicy to use for remote certificate
+        /// validation. If set, CertificateRevocationCheckMode and SslCertificateTrust is ignored.
+        /// </summary>
+        public X509ChainPolicy? ValidationPolicy { get; set; }
```

Comment:
Properties are supposed to have both a summary and a value tag.

https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value

```suggestion
        /// </summary>
        /// <value>
        ///   An optional customized policy for certificate chain validation.
        ///   The default is <see langword="null" />.
        /// </value>
        public X509ChainPolicy? ValidationPolicy { get; set; }
```